Changelog.md

```diff
@@ -13,6 +13,8 @@ Bug Fixes:
 * Fix `ActiveRecord::TestFixture#uses_transaction` by using example description
   to replace example name rather than example in our monkey patched
   `run_in_transaction?` method.  (Stan Lo, #2495)
+* Read configuration options from files to respect spec path setting.
+  (@vivekmiyani, #2508)
 
```

Comment:
```suggestion
```

This isn't what this PR does, instead you need a section on the enhancements section saying that generators now support this flag.